couch/tests/common.py

```diff
@@ -16,6 +17,8 @@
 
 HERE = os.path.dirname(os.path.abspath(__file__))
 
+COUCH_MAJOR_VERSION = int(re.split(r'\D+', os.getenv('COUCH_VERSION'))[0])
```

Comment:
I understand that the complexity comes from the docker images not being tagged the same but this seems overly complext for just extracting the MAJOR version. Can't we simply go with `int(os.getenv('COUCH_VERSION')[0])` here? I guess this will fail if couch starts to have two digits major version but we're probably very far from it right now

Reply:
Hum, `overly complexe` is relative, but I won't consider splitting by `r'\D+'` complexe :)